toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -1,6 +1,7 @@
 {
     "packages": [
         "pam",
+        "audit",
```

Comment:
@chalamalasetty can you please review this change

Reply:
I was initially concerned about image size increase from adding this package. But adding it to the iso initrd should be fine - I do not believe the core image would be affected.